lib/fathead/sass/parse_functions.py

```diff
@@ -0,0 +1,325 @@
+        Args:
+            section: A section of parsed HTML that represents a function definition
+        Returns:
+            A div element
+        """
+        anchor = self.parse_for_anchor(section)
+        heading = self.soup_data.find(id=anchor[1:])
+        return heading.parent
+
+    def parse_for_example(self, section):
+        info = self.parse_for_section_div(section)
+        example = info.find('div', {'class': 'examples'})
+        if example:
+            code = example.find('pre')
+            text = '<h5>Examples</h5>'+ str(code)
+            text = '\\n'.join(text.split('\n'))
+            return text
+        return None
+    
+    def parse_for_parameters(self, section):
+        info = self.parse_for_section_div(section)
+        parameters = info.find('ul', {'class': 'param'})
+        if parameters:
+            code = self.fix_parameter_links(parameters)
+            text = '<h5>Parameters</h5><ul>'
+            code = code.find_all('li')
+            for parameter in code:
+                text = text + '<li>'
+                name = parameter.find('span', {'class':"name"})
+                if name:
+                    text = text + name.text
+                inline = parameter.find('div', {'class':"inline"})
+                if inline:
+                    inline = parameter.find('p')
+                    inline = str(inline)
+                    inline = inline.replace('’','&#39;')
+                    text = text + " - " + inline
+                text = text + '</li>'
+            text = text + '</ul>'
+        
+            return text
+        return None
+        
+    def fix_parameter_links(self, parameters):
+        for a in parameters.findAll('a'):
+            path = a['href']
+            a['href'] = a['href'].replace(a['href'],'http://sass-lang.com/documentation/Sass/Script/'+path)
+        return parameters
+    def create_url(self, anchor):
+        """
+        Helper method to create URL back to document
+        Args:
+            anchor: #anchor
+
+        Returns:
+            Full URL to function on the sass doc
+
+        """
+        return SASS_DOC_BASE_URL + anchor
+
+    def parse_for_data(self):
+        """
+        Main gateway into parsing the data. Will retrieve all necessary data elements.
+        """
+        data = []
+        functions = []
+        
+        for function_section in self.function_sections:
+            function = self.parse_for_function_name(function_section[0])
+            if function:
+                method_signature = self.parse_for_method_signature(function_section[0])
+                description = self.parse_for_description(function_section[1])
+                anchor = self.parse_for_anchor(function_section[0])
+                example = self.parse_for_example(function_section[0])
+                parameter = self.parse_for_parameters(function_section[0])
+                abstract = method_signature + "<br>" + description
```

Comment:
@VeerpalBrar Instead of using `<br>` let's wrap the `description` in a `<p>` tag, and lets reverse the order here.

Description first, then code snippet.

The HTML should look like:

```html
<p>$description_text</p>
<pre><code>$method_signature</code></pre>
```

Do you understand?